lib/virt_autotest/utils.pm

```diff
@@ -140,4 +140,11 @@ sub install_default_packages {
     zypper_call '-t in nmap iputils bind-utils', exitcode => [0, 4, 102, 103, 106];
 }
 
+sub upload_y2logs {
+    # Create and Upload y2log for analysis
+    assert_script_run "save_y2logs /tmp/y2logs.tar.bz2", 180;
+    upload_logs("/tmp/y2logs.tar.bz2");
```

Comment:
bz2 is the default file format by save_y2logs ?

Reply:
refer to source code of save_y2logs  tool, which was supported the following 
compressed mode
```
  if [ "$(type -p xz)" ] ; then
    SUFFIX=.tar.xz
    COMPRESSION=xz
  elif [ "$(type -p gzip)" ] ; then
    SUFFIX=.tar.gz
    COMPRESSION=gzip
  elif [ "$(type -p bzip2)" ] ; then
    SUFFIX=.tar.bz2
    COMPRESSION=bzip2
  else
    SUFFIX=.tar
    COMPRESSION=
  fi
```
Generally, used with bzip2 as the recommended compressed mode.

Reply:
Suggest to use .tar.gz, which is more widely used and our tests mostly use this.

Reply:
Although bz2 is not so widely used, but tar -xz can unzip it correctly. BTW bz2 uses less log spaces . Let's keep it.